charts/vald/README.md

```diff
@@ -845,7 +845,7 @@ Please run the following command to install the chart with this values yaml,
     $ helm repo add vald https://vald.vdaas.org/charts
     $ helm install --values values-agent-ngt-standalone.yaml vald-agent-ngt vald/vald
 
-If you'd like to access the agents from out of the Kubernetes cluster, it is recommended to create an [Ingress][k8s-ingress].
+If you'd like to access the agents from out of the Kubernetes cluster, it is recommended to create an [Ingress][kubernetes-ingress].
```

Comment:
😇

Reply:
thank you for your review.
I will fix it soon.